tutor/specs/components/navbar/__snapshots__/support-menu.spec.jsx.snap

```diff
@@ -150,22 +155,28 @@ exports[`Support Menu renders support links when in a course for student 1`] = `
         </span>
       </a>
     </li>
-    <li
-      className="support-document-link"
-      role="presentation"
-      style={undefined}
+    <div
+      className="tour-anchor"
+      data-tour-anchor-id="menu-support-document"
+      id="menu-support-document"
     >
-      <a
-        href="https://d3bxy9euw4e147.cloudfront.net/oscms-prodcms/media/documents/openstax_tutor_student_getting_started_guide.pdf"
-        onClick={[Function]}
-        onKeyDown={[Function]}
-        role="menuitem"
-        tabIndex="-1"
-        target="_blank"
+      <li
+        className="support-document-link"
+        role="presentation"
+        style={undefined}
       >
```

Comment:
It looks to me like you've got an incorrect DOM here,  it's nesting as:
```
<ul>
    <div>
          <li>
```


I found where this is happening and noted it below